src/data/data.JSON

```
@@ -14013,7 +14013,7 @@
       "label": "components"
     },
     "js": {
-      "data": "jQuery",
+      "data": "Jquery",
```

```suggestion
      "data": "jQuery",
```

^ This is the correct capitalization

---

@castastrophe `data.json` is generated when we run the `npm run data-build`. This data is actually coming from `src/data/systems/201911111600-suse.json` that we haven't made any change. Maybe this Is because the data.json was not up to date. I recommend keeping it intact so that the project maintainer can generate the entire data if he wants.